packages/drawer/rc/__specs__/index.spec.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+
+import { Drawer } from '../index.js'
+
+describe('Drawer', () => {
+  describe('when the arrow icons is clicked', () => {
+    it('toggles open/close', async () => {
+      const { getByTestId } = render(
+        <Drawer>
+          <Drawer.Head data-testid="head">
+            <p>Click me to open</p>
+          </Drawer.Head>
+          <Drawer.Body data-testid="body">
+            <p>Drawer Content here</p>
+          </Drawer.Body>
+        </Drawer>
+      )
+
+      const btn = getByTestId('head')
+      const contentWrapper = getByTestId('body')
+      expect(contentWrapper).toHaveAttribute('aria-hidden', 'true')
+      fireEvent.click(btn)
+      expect(contentWrapper).toHaveAttribute('aria-hidden', 'false')
+      fireEvent.click(btn)
+      expect(contentWrapper).toHaveAttribute('aria-hidden', 'true')
+    })
+  })
+  const Controlled = () => {
```

Comment:
i'd say just move this component into the test because it's only used in one place